frontend/src/metabase/visualizations/components/TableInteractive/TableInteractive.jsx

```diff
@@ -115,26 +117,41 @@ class TableInteractive extends Component {
   static defaultProps = {
     isPivoted: false,
     hasMetadataPopovers: true,
-    renderTableHeaderWrapper: children => (
-      <div className={TableS.cellData} data-testid="cell-data">
-        {children}
-      </div>
-    ),
-    renderTableCellWrapper: children => {
-      const hasChildren = children != null && children !== "";
+    renderTableHeaderWrapper: (
+      children,
+      column,
+      columnIndex,
+      { theme } = {},
+    ) => {
       return (
-        <div
-          className={cx({
-            [TableS.cellData]: hasChildren,
-          })}
-          data-testid={hasChildren ? "cell-data" : undefined}
-        >
+        <Box className={TableS.cellData} data-testid="cell-data" c="brand">
```

Comment:
Diff here is a bit misleading. I moved `renderTableCellWrapper` to be a method rather than a prop. I checked and no one ever passes in this as a prop. The `data-testid` and `className` remains the same.